core/templates/services/UpgradedServices.ts

```diff
@@ -1074,6 +1079,15 @@ export class UpgradedServices {
     upgradedServices['LearnerDashboardBackendApiService'] =
       new LearnerDashboardBackendApiService(
         upgradedServices['HttpClient']);
+    upgradedServices['LearnerPlaylistBackendApiService'] =
```

Comment:
Are these needed here? If not, lets not add it

Reply:
Nope, Removed!